### DISCUSSION TITLE
logs and alerting, how ?

### DISCUSSION THREAD

**Discussion by StefanSa** [uptrace/uptrace#283] [Q&A]
Hi there,
First of all, thank you very much for Uptrace.
I am currently testing otel-contrib with windows eventlog.
I would now like to trigger an alarm for a specific event-id.
The field for this would be available `event_id.id`
Is there a chance to generate an alarm via this field, if so how?

Thanks for any help.
Stefan

**Comment by vmihailenco** [comment]
You can [create](https://uptrace.dev/get/span-metrics.html) a metric from logs:

```shell
metrics_from_spans:
  - name: uptrace.tracing.your_metric_name
    description: Logs with event_id.id
    instrument: counter
    unit: 1
    value: .count
    attrs:
      - .system
      - .group_id
    annotations:
      - display.name
    where: event_id.id = YOUR_ID
```

And then create a monitor for that metric in Alerts -> Monitors tab.
